.github/scripts/rebase-finish.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source `dirname $0`/config.sh
+
+REBASE_BRANCH=$1
+ORIGIN_BRANCH=$2
+
+git fetch --all --prune
+
+# Check if $REBASE_BRANCH exists.
+if ! git show-ref --verify --quiet refs/remotes/origin/$REBASE_BRANCH; then
+  echo "$REBASE_BRANCH does not exist!"
+  exit 1
+fi
+
+git switch $ORIGIN_BRANCH
+git reset --hard origin/$REBASE_BRANCH
+
+git config user.name github-actions
+git config user.email github-actions@github.com
+
+DAY=$(date +%Y-%m-%d)
```

Comment:
maybe hash is a better way

Reply:
Short hash of what? The commit hash is in the commit.

Reply:
yes, I was suggesting to add hash to backup branch name to prevent backup branch overwrite if it for some reason backup has been triggered during the same day